cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/release_test.go

```diff
@@ -708,6 +778,24 @@ func TestCreateInstalledPackage(t *testing.T) {
 			if err := mock.ExpectationsWereMet(); err != nil {
 				t.Errorf("there were unfulfilled expectations: %s", err)
 			}
+
+			// check expected HelmReleass CRD has been created
+			dynamicClient, _, err = s.clientGetter(context.Background())
+			if err != nil {
+				t.Fatalf("%+v", err)
+			}
+
+			releaseObj, err := dynamicClient.Resource(releasesGvr).Namespace("kubeapps").Get(
+				context.Background(),
+				tc.request.Name,
+				v1.GetOptions{})
+			if err != nil {
+				t.Fatalf("%+v", err)
+			}
+
+			if got, want := releaseObj.Object, tc.expectedRelease; !cmp.Equal(want, got) {
```

Comment:
Surprised you didn't need to ignore some fields... great, if so.

Reply:
yeah, me too. But it works, I tested it. Maybe these CRDs don't have any unexported fields?